### ISSUE TITLE
Multiple galleries on a single page

### ISSUE THREAD

**Issue by qbuben** [SchwarzwaldFalke/ngGallery#23]
Need to replace this code:

``` javascript
var $thumbwrapper = angular.element(document.querySelectorAll('.ng-thumbnails-wrapper'));
var $thumbnails = angular.element(document.querySelectorAll('.ng-thumbnails'));
```

With this:

``` javascript
var $thumbwrapper = element.find('.ng-thumbnails-wrapper');
var $thumbnails = element.find('.ng-thumbnails');
```

This allows to use multiple galleries on a single page and it should be faster to search element only inside scope.

**Comment by jkuri** [comment]
Hi. Can you submit a PR?

**Comment by qbuben** [comment]
No, sorry, i don't use github.

**Comment by SchwarzwaldFalke** [comment]
Fixed with #27

**Comment by SchwarzwaldFalke** [comment]
This PR introduced  #30 if jQuery isn't included.
The [angularJS documentation](https://docs.angularjs.org/api/ng/function/angular.element) says "find() - Limited to lookups by tag name" so this doesn't work with the class attribute.
